taskcluster/ci/browsertime/kind.yml

```diff
@@ -13,7 +13,7 @@ kind-dependencies:
 primary-dependency: signing
 
 only-for-build-types:
-    - nightly
+    - production
```

Comment:
Can you move this to the nightlySimulation build instead of production?

Reply:
Good news: This patch will rename `production` into `nightly` so this part won't change. Do you still want me to use `nightly-simulation`? (I don't have any preference for now)

Reply:
Yes please! We'll need to use nightly-simulation for sheriffing purposes.

Reply:
Oh! I just realized that if we change this to nightly-simulation then that would trigger it one every commit right? Then we'd have to make a cron for it.

I think it would be simpler for you to make the change to production and in the future when we get closer to sheriffing we'll make a cron and switch to the nightly-simulation build. How does that sound?